### PR TITLE
[MC][NET] Polish translation update

### PR DESCRIPTION
**Pull request by pithwz**
[NET_MSG.MC] Polish translation update: net.exe

### CODE REVIEW

File: sdk/include/reactos/mc/net_msg.mc [hunk 1]
```diff
@@ -2194,14 +2195,14 @@ NET SHARE sharename=drive:path [/GRANT:user,[READ | CHANGE | FULL]]
...
 Language=Polish
-NET SHARE sharename=drive:path [/GRANT:user,[READ | CHANGE | FULL]]
+NET SHARE nazwaudziału=dysk:ścieżka [/GRANT:użytkownik,[READ | CHANGE | FULL]]
                                [/USERS:number | /UNLIMITED]
-                               [/REMARK:"text"]
+                               [/REMARK:"tekst"]
                                [/CACHE:Manual | Documents| Programs | None]
-          sharename [/USERS:number | /UNLIMITED]
-                    [/REMARK:"text"]
+          nazwaudziału [/USERS:liba | /UNLIMITED]
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2202]:
```suggestion
          nazwaudziału [/USERS:liczba | /UNLIMITED]
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 2]
```diff
@@ -2194,14 +2195,14 @@ NET SHARE sharename=drive:path [/GRANT:user,[READ | CHANGE | FULL]]
           {sharename | devicename | drive:path} /DELETE
 .
 Language=Polish
-NET SHARE sharename=drive:path [/GRANT:user,[READ | CHANGE | FULL]]
+NET SHARE nazwaudziału=dysk:ścieżka [/GRANT:użytkownik,[READ | CHANGE | FULL]]
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2198]:
```suggestion
NET SHARE nazwa_udziału=dysk:ścieżka [/GRANT:użytkownik,[READ | CHANGE | FULL]]
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 3]
```diff
@@ -2194,14 +2195,14 @@ NET SHARE sharename=drive:path [/GRANT:user,[READ | CHANGE | FULL]]
...
-                               [/REMARK:"text"]
+                               [/REMARK:"tekst"]
                                [/CACHE:Manual | Documents| Programs | None]
-          sharename [/USERS:number | /UNLIMITED]
-                    [/REMARK:"text"]
+          nazwaudziału [/USERS:liba | /UNLIMITED]
+                    [/REMARK:"tekst"]
                     [/CACHE:Manual | Documents | Programs | None]
-          {sharename | devicename | drive:path} /DELETE
+          {nazwaudziału | nazwaurządzenia | dysk:ścieżka} /DELETE
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2205]:
```suggestion
          {nazwa_udziału | nazwa_urządzenia | dysk:ścieżka} /DELETE
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 4]
```diff
@@ -2303,35 +2304,47 @@ devicename         Is one or more printers (LPT1: through LPT9:)
...
-/CACHE:Manual      Enables manual client caching of programs and documents
-                   from this share.
-/CACHE:Documents   Enables automatic caching of documents from this share.
-/CACHE:Programs    Enables automatic caching of documents and programs
-                   from this share.
-/CACHE:None        Disables caching from this share.
+NET SHARE udostępnia zasoby serwera użytkownikom sieci. Użyte
+bez opcji wyświetla informacje o wszystkich zasobach udostępnionych
+na komputerze. Dla każdego zasobu, system ReactOS zgłasza
+nazwaurzadzenia lub nazwasciezki i powiązany z nim komentarz opisowy.
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2310]:
```suggestion
nazwa_urządzenia lub nazwa_ścieżki i powiązany z nim komentarz opisowy.
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 5]
```diff
@@ -2303,35 +2304,47 @@ devicename         Is one or more printers (LPT1: through LPT9:)
...
+
+nazwaudziału
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2312]:
```suggestion
nazwa_udziału       
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 6]
```diff
@@ -2303,35 +2304,47 @@ devicename         Is one or more printers (LPT1: through LPT9:)
...
+/USERS:liczba_użytkowników      
+  Ustawia maksymalną liczbę użytkowników, którzy mogą jednocześnie korzystać z udostępnionego zasobu
+
+/UNLIMITED
+  Określa nieograniczoną liczbę użytkowników, którzy mogą jednocześnie korzystać z udostępnionego zasobu.
+                   
+/REMARK:"tekst"
+  Dodaje opisowy komentarz dotyczący zasobu. Tekst należy wpisać w cudzysłowie.
+                   
+nazwaurzadzenia
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2331]:
```suggestion
nazwa_urządzenia
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 7]
```diff
@@ -3160,76 +3173,102 @@ Options      Are as follows:
 NET HELP command | MORE displays Help one screen at a time.
 .
 Language=Polish
-NET USER creates and modifies user accounts on computers. When used
-without switches, it lists the user accounts for the computer. The 
-user account information is stored in the user accounts database.
+NET USER tworzy i modyfikuje konta użytkowników na komputerach. 
+Stosowane bez przełączników, listy kont uzytkowników na komputerze. 
+Konto użytkownika informacje są przechowywane w bazie danych kont użytkowników.
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:3178]:
```suggestion
NET USER tworzy i modyfikuje konta użytkowników na komputerach. Użyte bez
przełączników wyświetla listę kont użytkowników na komputerze. Informacja
o kontach użytkowników przechowywana jest w bazie danych kont użytkowników.
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 8]
```diff
@@ -3160,76 +3173,102 @@ Options      Are as follows:
...
-             A password must satisfy the minimum length set with the 
-             /MINPWLEN option of the NET ACCOUNTS command. It can have as 
-             many as 14 characters. 
-*            Produces a prompt for the password. The password is not 
-             displayed when you type it at a password prompt. 
-/DOMAIN      Performs the operation on a domain controller of 
-             the current domain. 
-/ADD         Adds a user account to the user accounts database. 
-/DELETE      Removes a user account from the user accounts database. 
+nazwaużytkownika
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:3180]:
```suggestion
nazwa_użytkownika
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 9]
```diff
@@ -3160,76 +3173,102 @@ Options      Are as follows:
...
-                              Lists as many as eight computers from
-                              which a user can log on to the network. If 
-                              /WORKSTATIONS has no list or if the list is *,
-                              the user can log on from any computer.
+   /ACTIVE:{YES | NO}         Aktywuje lub dezaktywuje konto. Jeśli konto
+                              nie jest aktywne, użytkownik nie ma dostępu do
+                              serwera. Wartością domyślną jest YES.
+                              
+   /COMMENT:"tekst"           Opis konta użytkownika.
+                              Tekst musi być  ujęty w cudzysłów.
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:3213]:
```suggestion
                              Tekst musi być ujęty w cudzysłów.
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 10]
```diff
@@ -3160,76 +3173,102 @@ Options      Are as follows:
...
+                              wyrażony jako: dzień[-dzień][,dzień[-dzień]],
+                              czas[-czas],[,czas[-czas]], ograniczony jest do
+                              godzinnych przyrostów. Wpisz pełne lub skrócone
+                              nazwy dni tygodnia. Czas może być opisany
+                              w notacji 12- lub 24-godzinnej. Dla notacji
+                              12-godzinnej, użyj symbolu: am, pm, a.m. lub
+                              p.m. Parametr ALL oznacza, że użytkownik zawsze
+                              może się logować, brak wartości oznacza, że
+                              użytkownik nigdy nie może się logować.
+                              Oddziel  wpisy dnia i godziny przecinkiem;
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:3259]:
```suggestion
                              Oddziel wpisy dnia i godziny przecinkiem;
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 11]
```diff
@@ -3829,33 +3868,36 @@ The following conventions are used to indicate command syntax:
 .
 Language=Polish
 SYNTAX
-The following conventions are used to indicate command syntax:
+Do opisu składni poleceń użyto następujących zasad:
 
--  Capital letters represent words that must be typed as shown. Lower-
-   case letters represent names of items that may vary, such as filenames.
+-  Wielkich liter użyto do przedstawienia slów kluczowych, które trzeba
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:3873]:
```suggestion
-  Wielkich liter użyto do przedstawienia słów kluczowych, które trzeba
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 12]
```diff
@@ -3829,33 +3868,36 @@ The following conventions are used to indicate command syntax:
...
+   wpisać tak jak pokazano. Małych liter użyto do oznaczenia nazw elementów,
+   które są zmienne, np. nazw plików.
 
--  The [ and ] characters surround optional items that can be supplied
-   with the command.
+-  Elementy opcjonalnie występujące w poleceniu zawarto w znakach [ i ].
 
--  The { and } characters surround lists of items. You must supply one
-   of the items with the command.
+-  Listy elementów zawarto w znakach { i }, które oznaczaja, że
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:3879]:
```suggestion
-  Listy elementów zawarto w znakach { i }, które oznaczają, że
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 13]
```diff
@@ -3829,33 +3868,36 @@ The following conventions are used to indicate command syntax:
...
+   w poleceniu musi zostać zastosowany jeden z elementów listy
 
--  The | character separates items in a list. Only one of the items can
-   be supplied with the command.
+-  Znak | rozdziela elementy listy. Oznacza on, że w poleceniu może
+   zostać użyty tylko jeden z rozdzielanych elementów .
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:3883]:
```suggestion
   zostać użyty tylko jeden z rozdzielanych elementów.
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 14]
```diff
@@ -2303,35 +2304,47 @@ devicename         Is one or more printers (LPT1: through LPT9:)
...
-                   from this share.
-/CACHE:Documents   Enables automatic caching of documents from this share.
-/CACHE:Programs    Enables automatic caching of documents and programs
-                   from this share.
-/CACHE:None        Disables caching from this share.
+NET SHARE udostępnia zasoby serwera użytkownikom sieci. Użyte
+bez opcji wyświetla informacje o wszystkich zasobach udostępnionych
+na komputerze. Dla każdego zasobu, system ReactOS zgłasza
+nazwaurzadzenia lub nazwasciezki i powiązany z nim komentarz opisowy.
+
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2311]: What the following part looks like in CMD? Have you checked that?

Comment (pithwz) [reply sdk/include/reactos/mc/net_msg.mc:2311]: yes, copied from cmd 2k3. run cmd and type example: net help share

File: sdk/include/reactos/mc/net_msg.mc [hunk 15]
```diff
@@ -2303,35 +2304,47 @@ devicename         Is one or more printers (LPT1: through LPT9:)
...
-/CACHE:Programs    Enables automatic caching of documents and programs
-                   from this share.
-/CACHE:None        Disables caching from this share.
+NET SHARE udostępnia zasoby serwera użytkownikom sieci. Użyte
+bez opcji wyświetla informacje o wszystkich zasobach udostępnionych
+na komputerze. Dla każdego zasobu, system ReactOS zgłasza
+nazwa_urządzenia lub nazwa_ścieżki i powiązany z nim komentarz opisowy.
+
+nazwa_udziału       
+  Określa nazwę sieciową udostępnionego zasobu. Aby wyświetlić informacje dotyczące tylko danego zasobu, należy wpisać polecenie net share z parametrem
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2313]:
```suggestion
  Określa nazwę sieciową udostępnionego zasobu. Aby wyświetlić informacje dotyczące tylko danego zasobu, należy wpisać polecenie net share z parametrem.
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 16]
```diff
@@ -2303,35 +2304,47 @@ devicename         Is one or more printers (LPT1: through LPT9:)
...
+
+nazwa_udziału       
+  Określa nazwę sieciową udostępnionego zasobu. Aby wyświetlić informacje dotyczące tylko danego zasobu, należy wpisać polecenie net share z parametrem 
+
+dysk:ścieżka       
+  Określa ścieżkę absolutną katalogu do udostępnienia.
+
+/GRANT:user,perm
+  Tworzy udział z deskryptorem zabezpieczeń, który nadaje wymagane uprawnienia określonemu użytkownikowi.
+  Ta opcja może być używana wiecej niż raz w celu nadania uprawnień udziału wielu użytkownikom
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2320]:
```suggestion
  Ta opcja może być używana więcej niż raz w celu nadania uprawnień udziału wielu użytkownikom
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 17]
```diff
@@ -2303,35 +2304,47 @@ devicename         Is one or more printers (LPT1: through LPT9:)
...
+                   
+/USERS:liczba_użytkowników      
+  Ustawia maksymalną liczbę użytkowników, którzy mogą jednocześnie korzystać z udostępnionego zasobu
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2323]:
```suggestion
  Ustawia maksymalną liczbę użytkowników, którzy mogą jednocześnie korzystać z udostępnionego zasobu.
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 18]
```diff
@@ -2303,35 +2304,47 @@ devicename         Is one or more printers (LPT1: through LPT9:)
...
+
+/UNLIMITED
+  Określa nieograniczoną liczbę użytkowników, którzy mogą jednocześnie korzystać z udostępnionego zasobu.
+                   
+/REMARK:"tekst"
+  Dodaje opisowy komentarz dotyczący zasobu. Tekst należy wpisać w cudzysłowie.
+                   
+nazwa_urządzenia
+  Jest jedną lub kilkoma drukarkami (LPT1: do LPT9:)
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:2332]:
```suggestion
  Jest jedną lub kilkoma drukarkami (LPT1: do LPT9:).
```

File: sdk/include/reactos/mc/net_msg.mc [hunk 19]
```diff
@@ -3829,33 +3868,36 @@ The following conventions are used to indicate command syntax:
...
+   które są zmienne, np. nazw plików.
 
--  The [ and ] characters surround optional items that can be supplied
-   with the command.
+-  Elementy opcjonalnie występujące w poleceniu zawarto w znakach [ i ].
 
--  The { and } characters surround lists of items. You must supply one
-   of the items with the command.
+-  Listy elementów zawarto w znakach { i }, które oznaczają, że
+   w poleceniu musi zostać zastosowany jeden z elementów listy
```
Comment (archeYR) [inline sdk/include/reactos/mc/net_msg.mc:3880]:
```suggestion
   w poleceniu musi zostać zastosowany jeden z elementów listy.
```

**Comment by learn-more** [comment]
Please refrain from doing 30 commits in a row in the PR...

**Pull request merged**